cmd/antrea-controller/options.go

```diff
@@ -16,15 +16,26 @@ package main
 
 import (
 	"errors"
+	"fmt"
 	"io/ioutil"
+	"net"
+	"strings"
 
 	"github.com/spf13/pflag"
 	"gopkg.in/yaml.v2"
+	netutils "k8s.io/utils/net"
 
 	"antrea.io/antrea/pkg/apis"
 	"antrea.io/antrea/pkg/features"
 )
 
+const (
+	defaultClusterCIDRs         = "172.18.0.0/16"
+	defaultServiceCIDR          = "172.19.0.0/16"
```

Comment:
I'm a little surprised that we provide default value for clusterCIDR and serviceCIDR. Is it common to have it? At least this default service CIDR won't work for most clusters. I feel we should leave them empty and validate clusterCIDR must be set correctly when enabling nodeIPAMController.
@salv-orlando @jianjuns @antoninbas thoughts?

Reply:
I agree with @tnqn. The current solution is a bit too error prone. It's too easy for the user to enable Antrea NodeIPAM but forget to update these CIDRs, in which case they would just be left with a non-functional cluster. It's better to leave them empty by default and fail early if they are not provided.

Reply:
OK

Reply:
I agree to make them mandatory if NodeIpamController is enabled.
It's easier to not have any pod running rather than having pods running but with unwanted IPs, and then having to restart all of them

Reply:
I'll make CIDRs mandatory when IPAM is enabled, and mask sizes according to CIDR configs (IP4 when IP4 CIDRs are configured, same for 6).